content/ja/integrations/kubernetes_state_core.md

```diff
@@ -70,18 +70,18 @@ datadog:
 `kubernetes_state_core` のチェックを有効にするには、DatadogAgent リソースの設定 `spec.features.kubeStateMetricsCore.enabled` を `true` に設定する必要があります。
 
 ```yaml
-apiVersion: datadoghq.com/v1alpha1
 kind: DatadogAgent
+apiVersion: datadoghq.com/v2alpha1
 metadata:
   name: datadog
 spec:
-  credentials:
-    apiKey: <DATADOG_API_KEY>
-    appKey: <DATADOG_APP_KEY>
   features:
     kubeStateMetricsCore:
       enabled: true
-  # (...)
+  global:
+    credentials:
+      apiKey: <DATADOG_API_KEY>
+      appKey: <DATADOG_APP_KEY>
 ```
 
 注: Datadog Operator v0.7.0 以降が必要です。
```

Comment:
Not sure what this actually says, but should we bump the version to v1.0.0?

Reply:
yeah, according to google translate that is correct. I am fixing now but the JP team should review after too.